common/laws/00_citizenship.txt

```diff
@@ -227,6 +227,17 @@ law_cultural_exclusion = {
 			limit = { ai_has_enact_weight_modifier_journal_entries = yes }
 			add = 750
 		}
+		if = {
+			limit = { 
+				OR = {
+					has_law = law_type:law_ethnostate
+					has_law = law_type:law_national_supremacy
+					has_law = law_type:law_racial_segregation
+				}
```

Comment:
The journal entry needs to reflect that. Currently both multiculturalisms are not valid for completion of the journal entry.

Reply:
Why do I need to change this? It's for cultural exclusion.